tests/wpt/meta/cors/cors-safelisted-request-header.any.js.ini

```diff
@@ -15,10 +15,10 @@
     expected: FAIL
 
   [No preflight for {"range":"bytes=100-200"}]
-    expected: FAIL
+    expected: PASS
```

Comment:
Can these be removed instead of being marked as PASS?

Reply:
@jdm what do you think about?

Reply:
It's cleaner to remove them instead of changing FAIL to PASS.